rdr_service/dao/physical_measurements_dao.py

```diff
@@ -974,3 +980,20 @@ def _restore_record(self, record, resource):
 
         record = self.store_record_fhir_doc(record, doc)
         return record
+
+    @classmethod
+    def update_measurement_core_data_flag(cls, measurement_collection: PhysicalMeasurements):
+        """
+        Analyzes the physical measurement data to determine if the Core Data flag should be set to True
+        """
+        has_height = False
+        has_weight = False
+        for measurement in measurement_collection.measurements:
+            # Measurement should count if there is a value recorded, or if there are modifications/qualifiers
+            is_valid_value = measurement.valueDecimal is not None or measurement.qualifiers
+            if measurement.codeValue == 'height':
```

Comment:
Believe we'll need to account for the `pre-pregnancy-weight` code here as well (counts as a valid weight?).  For older PM records (e.g., when backfilling the new column) , the` loinc` system codes `8302-2` (height) and `29463-7` (weight) may be the only ones in the `measurement` data instead.

Reply:
Thanks. I wasn't aware of those codes. I'll get it updated.